python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-api-models.j2

```diff
@@ -8,16 +8,14 @@
 !
 management api models
 {%     for provider in management_api_models.providers | arista.avd.natural_sort('name') %}
-{%         if provider.name is arista.avd.defined and provider.name | lower in ['smash', 'sysdb'] %}
```

Comment:
Small change as already added valid_values under management_api_models.providers[].name and it is a primary key.

Reply:
I dont see the name as primary key in schema, can you please recheck.

Reply:
`name` is not marked as primary key. I removed natural sort on `name` and added arista.avd.defined check for `name` and `management_api_http.providers[].paths[].path`.

Reply:
Is is a bug that name is not a primary key, or could the same provider name be in EOS multiple times?